README.md

```diff
@@ -23,6 +23,11 @@ Optionally you can pass an `output` flag. This will save the jwt in the provided
 go run ./cmd/jwt/*.go -audience=https://issuer.zitadel.ch -key=key.json -output=jwt.txt
 ```
 
+You can also create a JWT by providing a RSA private key (.pem file). You then also need to specify the issuer of the token:
+```zsh
+go run ./cmd/jwt/*.go -audience=https://issuer.zitadel.ch -key=key.pem -issuer=client_id
```

Comment:
Should we update the domains here? maybe its missleading if there is the old domain